x-pack/plugins/apm/public/components/fleet_integration/apm_enrollment_flyout_extension.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiButton, EuiText, EuiSpacer } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
+import { AgentEnrollmentFlyoutFinalStepExtension } from '../../../../fleet/public';
+
+export function getApmEnrollmentFlyoutStepProps(): AgentEnrollmentFlyoutFinalStepExtension['stepProps'] {
+  // TODO: Figure out how to get `http.basePath` here
+  const installApmAgentLink = '/app/home#/tutorial/apm';
```

Comment:
How can I get Kibana's `basePath` value here without using a React hook?

Reply:
You would have to pass it in as an argument.

Reply:
I think it's a good argument to change the extension to have a title and a component, you will be able to use hooks in the component, and I am sure in the future it will prevent some headache to be able to use hooks here too. what do you think?

Reply:
Yeah that should work here. I have the refactor title + Component in the extension working locally. I will try pulling in hooks to the component tree rendered here to get at this data.